.circleci/config.yml

```
@@ -85,14 +85,47 @@ jobs:
           path: coverage
 
   test-rendering:
-    executor: browser-node
+    working_directory: ~/dist-flow-charts
+    docker:
+      - image: circleci/node:16.10.0-browsers
+      - image: mcr.microsoft.com/playwright:focal
     steps:
-      - use_workspace
-      - run: MOCHA_FILE=test-results/rendering/test.xml npm run test:rendering:ci
+      - attach_workspace:
+          at: ~/dist-flow-charts
```

I think could also be
```suggestion
      - use_workspace
```

should do the same thing.
but as it just saves one line. I am unsure if the reusable use_workspace command is worth keeping
(also apply to the same thing in test-integration)